examples/springboot/README.md

```diff
@@ -0,0 +1,93 @@
+# Spring Boot Sample Application for JavaBrake
+
+## About the application
+
+The example application provides three GET endpoints:
+
+1. `/date` - returns server date and time.
+2. `/locations` - returns list of available locations.
+3. `/weather/{locationName}` - returns the weather details for the locations.
+
+## Steps to run the API
+
+1. Install the dependencies for the application
+
+    ```
+    mvn install
+    ```
+
+2. You must get both `project_id` & `project_key`.
+
+    Find your `project_id` and `project_key` in your Airbrake account and replace them in `application.properties` file.
+
+    ```java
+    airbrake.project.id=project_id
+    airbrake.project.key=project_key
+    ```
+3. Need to remove/exclude other logger dependencies.
+
+    In pom.xml -
+
+    ```xml
+    <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-web</artifactId>
+        <exclusions>
+            <exclusion>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-logging</artifactId>
+            </exclusion>
+        </exclusions>
+    </dependency>
+    ```
+
+    And add following dependency
+
+    ```xml
+    <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-log4j2</artifactId>
+    </dependency>
+    ```
+
+    In gradle -
+
+    ```
+    implementation ('org.springframework.boot:spring-boot-starter-web')
+    {
+    exclude group: 'org.springframework.boot', module: 'spring-boot-starter-logging'
+    }
+    ```
+
+    And add following dependency
+
+    ```
+    implementation 'org.springframework.boot:spring-boot-starter-log4j2:2.7.3'
+    ```
+
+4. Run the application
```

Comment:
Is there a more specific command that can be provided to the user? Is there an available `mvn` target that can be used to quickly get up and running?

Reply:
As per my knowledge there is no command for quick setup.